src/test/java/edu/kpi/testcourse/property_tests/AddGetTests.java

```diff
@@ -0,0 +1,70 @@
+package edu.kpi.testcourse.property_tests;
+import edu.kpi.testcourse.dataservice.DataService;
+import edu.kpi.testcourse.dataservice.UrlAlias;
+import edu.kpi.testcourse.dataservice.User;
+import edu.kpi.testcourse.urlservice.AliasInfo;
+import edu.kpi.testcourse.urlservice.UrlService;
+import static org.quicktheories.generators.SourceDSL.strings;
+import io.micronaut.context.BeanContext;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import javax.inject.Inject;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.quicktheories.QuickTheory.qt;
+
+@MicronautTest
+public class AddGetTests {
+  @Inject DataService dataService;
+  @Inject UrlService urlService;
+
+  @BeforeEach
+  void beforeEach() {
+    dataService.clear();
+    dataService.addUser(new User("jusovch", "j2202"));
+  }
+
+  @Test
+  void testUrlServiceAddGet() {
+    qt()
+      .forAll(
+        strings().basicLatinAlphabet().ofLengthBetween(4,10),
+        strings().basicLatinAlphabet().ofLengthBetween(4,10)
+      ).check((alias, url) -> {
+      url = url.concat(".com");
```

Comment:
Why have you put concat here? To make it look more like real url?

Reply:
To generate the URL in the appropriate form "[randomly chosen symbols].com"'